src/fsharp/TypedTreeOps.fs

```diff
@@ -5142,34 +5142,40 @@ let freeInModuleOrNamespace opts mdef =
 //---------------------------------------------------------------------------
 
 let rec stripLambda (expr, ty) = 
-    match expr with 
+    match stripDebugPoints expr with 
```

Comment:
These will run only with `--optimize+`, right?

Reply:
Nice ... I saw this last night, which is why the il compare test is commented out, but it was getting late and I needed my beauty sleep.